Makefile

```diff
@@ -96,8 +96,9 @@ check-all: lint check-static check-shadow check-gosec staticcheck errcheck
 check-setup:
 	@which retool >/dev/null 2>&1 || go get github.com/twitchtv/retool
 	@GO111MODULE=off retool sync
+	@which terraform >/dev/null 2>&1 || go get github.com/hashicorp/terraform@v0.12.12
```

Comment:
better to install terraform in hack/check-terraform.sh script if 1) terraform is not installed 2) terraform version is not v0.12.12

in practice, it is also better to install it in a private place (do not touch global installed one). this can be improved when we have a private binary directory.